### PR TITLE
Add administer blocks permission to site manager

### PR DESCRIPTION
**Pull request by jacob-korf**
Closes https://github.com/CuBoulder/tiamat-theme/issues/1099.
Adds the administer blocks permission, as that is layout manager's only permission to the site manager.

### CODE REVIEW

**Review by patrickbrown-io — APPROVED** [review]
Works as advertised!

**Comment by patrickbrown-io** [comment]
@jcsparks said to hold off until talking to the UI/UX team to make sure they're okay with it. Going to tag him for review as well but once we get the UI/UX approval this is good to merge

**Comment by jcsparks** [comment]
Got confirmation from Kevin and Nicole.  Merging in now.

**Review by jcsparks — APPROVED** [review]

**Pull request merged**
